UICatalog/Scenarios/AllViewsTester.cs

```diff
@@ -348,9 +348,9 @@ private View CreateClass (Type type)
             view.ColorScheme = Colors.ColorSchemes ["Base"];
         }
 
-        if (view.GetType().GetInterface ("IDesignable") is { } iface)
+        if (view is IDesignable iface)
```

Comment:
This check is always true. The explicit cast on line 343 makes it that way.

Reply:
Not getting what you're putting down here. If the View created on line 343 doesn't implement `IDesignable` the if statement on line 351 is false.

Reply:
I'll look again when I'm back at my PC later. Could have been a brain fart. 🤷‍♂️